### ISSUE TITLE
Bootloop to Recovery after installing Magisk 19.3 on my Pixel 2 XL running Security Patch July 2019

### ISSUE THREAD

**Issue by Wreck5tep** [topjohnwu/Magisk#1660]
Hey!
Like it says in the title: If I flash Magisk 19.3 using TWRP 3.3.0-0 on my Pixel 2 XL it doesn't start up anymore until I restore the stock boot image. 
It gets stuck on the Google Logo for like 2mins and then reboots straight into recovery.

The Stock Image I'm using is 9.0.0 (PQ3A.190705.001, Jul 2019).
It worked on last month's update just fine (PQ3A.190605.003, Jun 2019).

Hope you can fix this, and thanks for the great work!

**Comment by topjohnwu** [comment]
I believe you have some incompatible modules installed.....
Remove the modules in custom recovery, or simply use the uninstaller to completely remove Magisk

**Comment by Wreck5tep** [comment]
Oh wow that actually solved the issue, thank you very much!
I didn't think of doing that because I did a clean flash by using the flash-all.bat but I guess that didn't clean up Magisk.

**Comment by chrstfer** [comment]
I'm having the same issue on my pixel 2, with the same (non-xl) versions, but the suggested fix does not work. Running the uninstaller removes magisk and I can boot, but even after I've cleaned it, I still get the bootloop if I install magisk again (with twrp or with the patched img via adb). Same thing with the beta version of magisk too. 

I know this is a closed ticket but the fix didn't work even though my issue seemed so similar to this one that I figured I should duplicate it. If I should start a new issue let me know.
